parse/storage.go

```diff
@@ -0,0 +1,28 @@
+package parse
```

Comment:
I like the idea of moving this out of the `cmd/` tree. But can we rename this package to just `cli` to match https://github.com/micromdm/nanomdm/commit/ff8b7af3437525755d0d6b05d3c710590b454b6a?